packages/business-rules/src/schemas/householder-appeal/update.js

```diff
@@ -12,22 +14,30 @@ const update = pinsYup
     id: pinsYup.string().uuid().required(),
     horizonId: pinsYup.string().trim().max(20).nullable(),
     lpaCode: pinsYup.string().trim().max(20).required(),
+    appealType: pinsYup.lazy((appealType) => {
+      if (appealType) {
+        return pinsYup.string().oneOf(Object.values(APPEAL_ID));
+      }
+      return pinsYup.string().nullable();
+    }),
     decisionDate: pinsYup.lazy((decisionDate) => {
       return pinsYup
         .date()
         .isInThePast(decisionDate)
-        .isWithinDeadlinePeriod(decisionDate)
+        .test('decisionDate', 'decisionDate is not within dealine period', function test() {
+          return (
+            appeal.decisionDate.isWithinDecisionDateExpiryPeriod(
```

Comment:
If `isWithinDecisionDateExpiryPeriod` is defined in `src/lib/pins-yup.js` then it would be a method of 'pinsYup' and wouldn't need to be imported separately here.

Reply:
Reinstated